common/org.testeditor.dsl.common.ui/src/org/testeditor/dsl/common/ui/utils/ProjectContentGenerator.xtend

```diff
@@ -329,6 +329,7 @@ class ProjectContentGenerator {
 			
 					<xtext.version>2.9.0</xtext.version>
 					<xtend.version>${xtext.version}</xtend.version>
+					<project.version>1.0.0-SNAPSHOT</project.version>
```

Comment:
`project.version` is already a Maven property that is determined by the Maven build.
The property should be called `testeditor.version` and referenced below so that we can configure this separately.